File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Internal/Marshall/MarshallingContextState.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal;

namespace Db4objects.Db4o.Internal.Marshall
{
	/// <exclude></exclude>
	public class MarshallingContextState
	{
		internal readonly MarshallingBuffer _buffer;

		public MarshallingContextState(MarshallingBuffer buffer)
		{
			_buffer = buffer;
		}
	}
}

File: db4oj/core/src/com/db4o/internal/marshall/MarshallingContextState.java
/* Copyright (C) 2007  Versant Inc.  http://www.db4o.com */

package com.db4o.internal.marshall;

import com.db4o.internal.*;


/**
 * @exclude
 */
public class MarshallingContextState {
    
    final MarshallingBuffer _buffer;

    public MarshallingContextState(MarshallingBuffer buffer) {
        _buffer = buffer;
    }

}
